rswag-specs/lib/rswag/specs/example_group_helpers.rb

```diff
@@ -149,6 +154,19 @@ def run_test!(description = nil, *args, **options, &block)
           end
         end
       end
+
+      private
+
+      def _parse_metadata(metadata)
+        metadata.reduce({}) do |acc, item|
+          if item.is_a?(Symbol)
+            acc[item] = true
+          elsif item.is_a?(Hash)
+            acc.merge!(item)
+          end
+          acc
+        end
+      end
```

Comment:
Mini nitpick - this avoids the need for returning something from the block: 🤷🏼 

```suggestion
      def _parse_metadata(metadata)
        metadata.each_with_object({}) do |item, acc|
          if item.is_a?(Symbol)
            acc[item] = true
          elsif item.is_a?(Hash)
            acc.merge!(item)
          end
        end
      end
```